CAP/gap/Finalize.gi

```diff
@@ -162,8 +165,14 @@ InstallMethod( IsFinalized,
             if ForAll( current_final_derivation.can_compute, j -> CurrentOperationWeight( weight_list, NameFunction( j[ 1 ] ) ) < infinity ) and
               ForAll( current_final_derivation.cannot_compute, j -> CurrentOperationWeight( weight_list, NameFunction( j ) ) = infinity ) then
                 
-                Add( current_installs, i );
-                
+                current_tester_func := current_final_derivation.category_filter;
+
+                if IsFilter( current_tester_func ) and Tester( current_tester_func )( category ) and current_tester_func( category ) then
```

Comment:
Is this also a bugfix of the previous version?
It appears to be the case that final derivations where tried to be executed
regardless of the filters, is that correct?

Reply:
Indeed, that is the case. Do you think this can cause problems? It is indeed a silent bugfix ;)